src/components/LockExceptionsSubpageHeader/LockExceptionsSubpageHeader.js

```diff
@@ -0,0 +1,35 @@
+import i18n from '@dhis2/d2-i18n'
+import classnames from 'classnames'
+import FontIcon from 'material-ui/FontIcon'
+import PropTypes from 'prop-types'
+import React from 'react'
+import { Link } from 'react-router-dom'
+import styles from './LockExceptionsSubpageHeader.module.css'
+
+const LockExceptionsSubpageHeader = ({ title }) => (
```

Comment:
This component renders this part of the lock exceptions subpages:
![image](https://user-images.githubusercontent.com/4295266/123801849-8fb2cb80-d8e2-11eb-90a9-b3edb886e02b.png)

It was extracted from https://github.com/dhis2/data-administration-app/blob/master/src/pages/lock-exception/LockException.js#L78-L103